src/sql/workbench/browser/modelComponents/radioCardGroup.component.ts

```diff
@@ -147,7 +147,7 @@ export default class RadioCardGroup extends ComponentBase<azdata.RadioCardGroupC
 		super.setProperties(properties);
 		// This is the entry point for the extension to set the selectedCardId
 		if (this.selectedCardId) {
-			this.selectCard(this.selectedCardId);
+			this._changeRef.detectChanges();
```

Comment:
please explain why the call to selectCard is removed?

Reply:
and please test the first launch experience to see whether a default card is set properly, I believe the selectcard will trigger an event, make sure missing this event won't cause any side effect.

Reply:
I made this change because I feel if the extension is making that change then we do not have to generate an event.

Reply:
Is this related to the other changes in this PR at all? If this isn't something you need for your changes please revert and submit a separate PR so we can discuss it more thoroughly.

Reply:
I am adding this back. Will create a separate PR if needed